test/mocks/context_mock.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+
+class CredsMock():
```

Comment:
One question. Did you look for a mock library FactoryGirl style?

Reply:
Nop